docs/reference/migration/migrate_5_0/settings.asciidoc

```diff
@@ -172,3 +172,18 @@ Two cache concurrency level settings
 `indices.fielddata.cache.concurrency_level` because they no longer apply to
 the cache implementation used for the request cache and the field data cache.
 
+==== Using system properties to configure Elasticsearch
+
+Elasticsearch can be configured by setting system properties on the
+command line via `-Des.name.of.property=value.of.property`. This will be
+removed in a future version of Elasticsearch. Instead, use
+`-E es.name.of.setting=value.of.setting`. Note that in all cases the
```

Comment:
Yes, and there is a [test](https://github.com/jasontedor/elasticsearch/commit/8a05c2a2bef6b5980df286d885ae732657872d15#diff-179c30a6dc225a58837c95771941fbc1R124) that shows this. :)